packages/astro/src/vite-plugin-head-propagation/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { AstroSettings } from '../@types/astro';
+import type { AstroPluginMetadata } from '../vite-plugin-astro/index';
+import type { ModuleInfo } from 'rollup';
+
+import * as vite from 'vite';
+
+const injectExp = /^\/\/[ ]*astro-head-inject/;
```

Comment:
```suggestion
const injectExp = /^\/\/\s*astro-head-inject/;
```

I think we can use `\s` for a space here